doc/notes/11455_deployments/11455_export_depl.dnt.adoc

```diff
@@ -0,0 +1,227 @@
+
+==== 5.2.2 WASL descriptive files
+
+WASL build sets have a descriptive file named after the project key letters.  The file has an extension of dmp.
+
+This description contains build set information comprised of in place information as well as pointers to other files.  The other files contain further build set information, and in some cases the actual bridge implementations.
+
+==== 5.2.2.3 Build set/project information
+
+```
+$BUILD_SET_NAME <project.name>
+$BUILD_SET_VERSION 0
+$PROJECT_KEY_LETTER <project.keyletters>
+$PROJECT_VERSION 0
+```
+
+Added templates:
+
+* t.project_begin.wasl
+* t.project_end.wasl
+
+A new operation is added to project, render_wasl.  The genfile.render_wasl operation is updated to move all project related logic to the new operation.
+
+Calls to the two templates are added to render_wasl.  The project_end is output after all required description data is output.  For all cases below that are output to the description file, they are output inbetween the start and end project templates.
+
+==== 5.2.2.4 Build set tags
+
+Tags will not be used, so this will always be empty.
+
+Added templates:
+
+* t.build_set_tags_begin.wasl
+* t.build_set_tags_end.wasl
+
+```
+$START_BUILD_SET_TAGS
+$END_BUILD_SET_TAGS
+```
+
+==== 5.2.2.5 Scenario list
+
+A pointer to a file containing all scenarios in the build set.
+
+Added template:
+
+* t.scenario_list_spec.wasl
+
+```
+$START_SCENARIO_LIST
+Events.scn
+$END_SCENARIO_LIST
+```
+
+==== 5.2.2.5.1 <project.keyletters>.scn file
+
+This file is output with an ordered list of sequenced terminators.  The list of these terminators is collected in the changes described in <<scenario_file>>.
+
+==== 5.2.2.6 Scenario list [[scenario_file]]
+
+An ordered list of scenarios that are executed in the build set.
+
+To support gathering the scenarios for output a new operation is added to project, render_scenario_list().  This operation will handle two things, collecting a filename list and collecting data according to the list definition below.  A new operation is also added to domain, render_scenario_definitions.  This operation handles production of the two collections required.  The data specified in the start scenarios tag is output in between the new templates below.
+
+Added templates:
+
+* t.scenario_list_begin.wasl
+* t.scenario_list_end.wasl
+
+```
+$START_SCENARIOS
+<project.name>,<project.version>,<project.name>::<activity.name>,<routine.number>,<project.name>::<activity.name>Scenario<routine.number>.scn
+$END_SCENARIOS
+```
+
+==== 5.2.2.7 Hand coded file pointer
+
+Added templates:
+
+* t.hcf.wasl
```

Comment:
There are hand-coded files in domain exports as well.  So, give this a name something like t.project_hcf.wasl.

Reply:
Done.